canonical/anbox-cloud.io
# Update index page title

## Done
Update the title for the index page to `Anbox Cloud - Scalable Android in the cloud`

## QA
- Open the demo and make sure the page title is now `Anbox Cloud - Scalable Android in the cloud`


## Issue / Card

Fixes #25
